tiledbcontents/tiledbcontents.py

```diff
@@ -242,7 +242,7 @@ def get_s3_credentials(namespace):
         else:
             organization = tiledb.cloud.client.organization(namespace)
             if organization.default_s3_path is not None:
-                return profile.default_s3_path_credentials_name
+                return organization.default_s3_path_credentials_name
```

Comment:
If the organization doesn't have a default credential for the s3 path set this returns None and then we end up falling back server side to the organizations default s3 credential right?

Reply:
This is correct, checked all possible scenarios